doc/release-notes.md

```diff
@@ -108,6 +108,15 @@ Updated RPCs
   Respectively, these new fields indicate the duration of a ban and the time remaining until a ban expires,
   both in seconds. Additionally, the `ban_created` field is repositioned to come before `banned_until`. (#21602)
 
+- The `testmempoolaccept` RPC now accepts multiple transactions (still experimental at the moment,
```

Comment:
In d93d51f1132bc6ae54f7d23a5bbb434ce01b7c37 "[doc] add release note for package testmempoolaccept"

nit: Generally release notes are added in a separate file named `release-notes-<pr number>.md` rather than `release-notes.md` to avoid causing rebase conflicts for other PRs. So for this PR, you would put this release note in `release-notes-20833.md`.

Reply:
Ah thank you, noted!

Reply:
Done